### DISCUSSION TITLE
html tags in event body

### DISCUSSION THREAD

**Discussion by glda19** [kewisch/gdata-provider#392] [Support] [answered]
Hi 
I use onecalendar on my android phone and when i fill in the event body in tb 91.5.1 and open the event in onecalendar  i see

i see html tags in the event body in google calendar  i don't have it
Is there not a newer version i tought there is a version 93 is that wright and wen released
![One Calendar](https://user-images.githubusercontent.com/45542607/152568438-9866241f-49dc-4d3b-ae77-289bb7b24ec3.jpg)

**Comment by ManuelBoss** [comment, marked as answer]
I have the same problem.
To solve, I'd try to enable the option "force use of plain text in event descriptions".
![grafik](https://user-images.githubusercontent.com/59780024/157084924-9622fb9b-1947-4cb8-809d-5f4a457e8ce8.png)
Unfortunately, the setting disables itself on exiting the options menu of the add-on automatically.
Is that working for you?

**Reply by kewisch** [reply]
The setting not being saved is fixed now :)
